test/parallel/test-assert.js

```diff
@@ -26,7 +26,7 @@ const a = assert;
 
 function makeBlock(f) {
   const args = Array.prototype.slice.call(arguments, 1);
-  return function() {
+  return () => {
```

Comment:
This makes `this` lexical. Is it OK here?

Reply:
Thank you for your review.

I think that it is OK because test passed locally.
It is not called with neither `.call` nor `.apply` in this file.
This code equivalent to:

```js
  return () => {
    return f.apply(null, args);
  };
```

Should I replace `this` with `null` ?

Reply:
Maybe, for a clarity, but let's see what others think)

Reply:
I got it.
Please let us know what others think.